src/FireStation/FireStation.cpp

```diff
@@ -62,15 +50,19 @@ namespace LockFreeDispatch {
         curNumCrew = newNumCrew;
     }
 
-    // Get curNumVehicles
-    uint8_t FireStation::getCurNumVehicles() const
+    FireStation::FireStation(uint32_t id, Location location, uint16_t maxCrew, uint16_t curCrew)
```

Comment:
```suggestion
    // earlier in the code
    using crew_size_t = uint16_t;
    using station_id_t = uint32_t;
    // ...
    FireStation::FireStation(station_id_t id, Location location, crew_size_t maxCrew, crew_size_t curCrew)
```
IMHO, one of the most beautiful aspects of C++ is its simple and extensible strong type system. Creating semantic aliases for primitive types adds semantic meaning to differentiate different uses of the same primitive type and better facilitates changing types later. See, for example: https://stackoverflow.com/a/4541362/2769271.